.github/workflows/tests.yml

```diff
@@ -2,8 +2,6 @@ name: EvaP Test Suite
 
 on:
   push:
-    branches:
-      - main
```

Comment:
Yes, you are right. I removed these lines to test the actions on the branch in my fork and forgot to restore it. Your point seems reasonable though

Reply:
I checked again: Translation commits are always pushed to main, so that already triggers a CI run, so we are fine with the old version. So I'd suggest just adding these two lines back.